app/assets/javascripts/analytics/ecommerce.js

```diff
@@ -0,0 +1,62 @@
+(function () {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  var Ecommerce = function (config) {
+    this.init = function (element) {
+      var self = this;
+      var ecommerceRows = element.find('[data-ecommerce-row]');
+      var startPosition = parseInt(element.data('ecommerce-start-index'));
+
+      ecommerceRows.each(function(index, ecommerceRow) {
+        var $ecommerceRow = $(ecommerceRow);
+
+        var contentId = $ecommerceRow.attr('data-ecommerce-content-id'),
```

Comment:
Minor: Is there a preference for `$ecommerceRow.attr('data-ecommerce-content-id')` over `$ecommerceRow.data('ecommerce-content-id')`? If the value of this data was altered the former wouldn't pick up the latest value.